src/main/wlp/server.xml

```diff
@@ -1,24 +1,24 @@
-<server description="Liberty beta">
+<server description="Liberty JTA Sample">
 
-<featureManager>
-<feature>localConnector-1.0</feature>
-<feature>servlet-3.1</feature>
-<feature>jsonp-1.0</feature>
-<feature>jdbc-4.1</feature>
-<feature>jndi-1.0</feature>
-<feature>cdi-1.2</feature>
-</featureManager>
+    <featureManager>
+        <feature>localConnector-1.0</feature>
+        <feature>servlet-3.1</feature>
+        <feature>jsonp-1.0</feature>
+        <feature>jdbc-4.1</feature>
+        <feature>jndi-1.0</feature>
+        <feature>cdi-1.2</feature>
+    </featureManager>
 
+	<httpEndpoint host="*" httpPort="9080" httpsPort="9443" id="defaultHttpEndpoint"/>
 
-<httpEndpoint host="*" httpPort="9080" httpsPort="9443" id="defaultHttpEndpoint"/>
+	<library id="DerbyLib">
+        <file name="${wlp.user.dir}/shared/resource/derby/derby.jar"/>
+    </library>
 
-<dataSource id="DefaultDataSource">
-<jdbcDriver libraryRef="DerbyLib"/>
-<properties.derby.embedded createDatabase="create" databaseName="${shared.resource.dir}/derby/JTASAMPLE"/>
-</dataSource>
-<library id="DerbyLib">
-<file name="${shared.resource.dir}/derby/derby.jar"/>
-</library>
+    <dataSource id="DefaultDataSource" jndiName="jtaDB">
+        <jdbcDriver libraryRef="DerbyLib"/>
+        <properties.derby.embedded createDatabase="create" databaseName="${wlp.user.dir}/shared/resource/derby/JTASAMPLE"/>
```

Comment:
change it back to 

`<properties.derby.embedded createDatabase="create" databaseName="${shared.resource.dir}/derby/JTASAMPLE"/>`